.gitignore

```diff
@@ -10,6 +10,7 @@
 *.log*
 .nyc_output
 *.aes
+*.code-workspace
```

Comment:
What's this doing?

Reply:
Oh sorry I should have added a comment, this allows people to use custom local workspace files, without seeing them in the repo diff.

I maintain two clones of the tupaia repo locally (`tupaia` and `tupaia2`). I use the default workspace file for the first (`tupaia-packages.code-workspace`), and I have added a `tupaia2.code-workspace` file in `tupaia2` so I can quickly toggle between them in VSCode 😄